generate_gt_from_box.py

```diff
@@ -0,0 +1,30 @@
+#!/usr/bin/env python3
+
+import argparse
+import io
+import unicodedata
+
+#
+# command line arguments
+#
+arg_parser = argparse.ArgumentParser('''Creates groundtruth files from text2image generated box files''')
+
+# Text ground truth
+arg_parser.add_argument('-t', '--txt', nargs='?', metavar='TXT', help='Line text (GT)', required=True)
+
+# Text box file
+arg_parser.add_argument('-b', '--box', nargs='?', metavar='BOX', help='text2image generated box file (BOX)', required=True)
+
+args = arg_parser.parse_args()
+
+#
+# main
+# uses "λ" for substitution to  get the space delimiters - change for Greek text which uses "λ"
```

Comment:
Why not use a control character outside like `NUL` or an dingbat/emoji instead of the fairly frequent lambda?